ArdusatSDK.h

```diff
@@ -315,10 +315,13 @@ class Orientation: public Sensor {
 
     boolean read(Acceleration & accel, Magnetic & mag);
 
-    const char * toCSV(const char * sensorName);
-    const char * toJSON(const char * sensorName);
+    const char * readToCSV(const char * sensorName);
```

Comment:
Have you tested `readToCSV` with the Orientation sensor? I believe you need to declare it `virtual` if you are overriding it (although it's possible the compiler is smart enough to do that implicitly...)